SU-SOE/engineering_profile
# SOED9-77: Magazine configs

# READY FOR REVIEW

# Summary
- This PR has the configs necessary for the additional fields and functionality for magazine stories on the engineering site.
- additionally, there are updates I've pulled in from the `soe_profile` which bring us up to date with that as well.

# Needed By (Date)
- asap

# Urgency
- medium

# Steps to Test

1. Check out this branch.
2. `drush cim` (it may need a `blt drupal:update` instead, since there's a bunch of changes)
3. Try to create a new News item; notice the addition of the new fields for SOE and for the magazine specifically.
4. verify nothing breaks.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
